app/web/features/communities/events/EventCard.tsx

```diff
@@ -125,7 +124,7 @@ export interface EventCardProps {
 
 export default function EventCard({ event, className }: EventCardProps) {
   const classes = useStyles({
-    eventImageSrc: event.photoUrl || eventImagePlaceholder,
+    eventImageSrc: event.photoUrl || "/img/eventImagePlaceholder.svg",
```

Comment:
Might be worth extracting the default value to an `eventImagePlaceholder` constant (rather than importing as before) since it's needed in a few places?